js/qmwp-add-measurement.js

```diff
@@ -583,14 +583,18 @@ jQuery(document).ready(function () {
                     variableValue = variable.mostCommonValue;
                 }
 
+                setInputValues(variableUnit, variableValue);
+
             }
 
-            if (variableUnit === null) {
-                return;
+
+            function setInputValues(unit, value) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['setInputValues' is defined but never used.](https://www.codacy.com/app/m_3/QuantiModo-WordPress-Plugin/file/1344935451/issues/source?bid=2836188&fileBranchId=2967214#l591)